src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -33,46 +32,46 @@ android {
 ```
 
 The Flutter engine has an `x86` and `x86_64` version. When using an emulator
-in debug Just-In-Time (JIT) mode, the Flutter module will still run correctly.
+in debug Just-In-Time (JIT) mode, the Flutter module still runs correctly.
 
 TODO(blasten): revise whether this is right.
 {{site.alert.end}}
 
 ## Using Android Studio
 
-The Android Studio IDE offers a convenient way of integrating your Flutter
+The Android Studio IDE is a convenient way of integrating your Flutter
```

Comment:
'is' may be a slightly more awkward word here. Android Studio has tons and tons of functions. One very minor part is the Flutter module integration mechanism. 

It's a bit like 'the car is a convenient way of getting around' vs 'the car is a convenient way of lighting your cigarette'.

Reply:
“Offers” is more of a marketing term. I didn’t think it belonged here, but if you think it does what you’re looking for, I have no objection.

Reply:
Perhaps 'The Android Studio IDE has a convenient way of integrating your Flutter module automatically using the Flutter plugin'?

Reply:
Yes!